frontend/coffee/edit_mod.coffee

```diff
@@ -26,7 +26,6 @@ window.upload_bg = (files, box) ->
         else
```

Comment:
I think `xhr.open('POST', "/api/mod/#{window.mod_id}/update-bg")` is being evaluated by the wrong route:

![image](https://user-images.githubusercontent.com/1559108/127689436-bbfd47ad-c4fe-41d0-8260-a29b4a22c3a3.png)

That text is only returned by `/api/mod/<int:mod_id>/<version>`:

https://github.com/KSP-SpaceDock/SpaceDock/blob/3197cf7b3d83fd33076e8a9e63ccc6fbb28e7286/KerbalStuff/blueprints/api.py#L408

Reply:
Yes, you are doing a GET request there, which won't hit the correct route.
If you do a POST by actually trying to upload a background image, it calls the correct function.
However, as you said, there's indeed a bug, in that it breaks if there was no image before and `old_path` is `None`.

Fix will be pushed in a minute :+1: